src/content/basics/app-platform/index.md

```diff
@@ -9,13 +9,23 @@ aliases:
   - /basics/app-catalog/
 owner:
   - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - What does App Platform do?
+  - What is an App Catalog?
+  - Does Giant Swarm provide app catalogs out of the box?
+  - How is App Platofrm implemented?
+  - What are the components of App Platform?
+  - What are the app catalogs that Giant Swarm provides out of the box?
+  - How can I create an organizational app catalog?
+  - How can I interact with the Giant Swarm App Platform?
+  - Can I create an app catalog?
```

Comment:
What is the different between this and "How can I create an organizational app catalog?" ?

Reply:
Good point. I was a bit torn between adding both. 
What ultimately won the day is that one is about the possibility and the other is how to undertake the task. Since these are search-related questions, I'm not completely sure a reader searching would default to how to undertake the task, since s/he might not be clear that it is even an option. If that makes sense.

Reply:
> What ultimately won the day is that one is about the possibility and the other is how to undertake the task.

OK cool I think I get this. So organizational app catalog Q is because we want to make this possible.

But I'm still seeing 3 references to control plane. I think these should also change to management cluster. Can you check again?

Reply:
Yes, those are still the references to Control Plane API. As I mentioned I am waiting on word from the WG to see what they ultimately decide to rename it to.

Reply:
Ah OK I missed that. Thanks for clarifying. 👍